src/models/callbacks/callback_right_view.py

```diff
@@ -0,0 +1,43 @@
+import os.path
+from pytorch_lightning.callbacks import Callback
+
+
+class RightCallback(Callback):
```

Comment:
minor: slightly prefer the previous name since it isn't intuitive what this does from the class name